feature/split/src/main/java/com/blitzsplit/split/presentation/viewmodel/SplitState.kt

```diff
@@ -0,0 +1,6 @@
+package com.blitzsplit.split.presentation.viewmodel
+
+data class SplitState(
+    val billTitleTextFieldValue: String = "",
+    val billValueTextFieldValue: String = ""
+)
```

Comment:
Sei que parece bobagem, mas a definição de primeiro estado é uma regra de negócio e não deveria ser feita pelo `data class` que representa o estado. Recomendo fazer isso na view model mesmo

```suggestion
data class SplitState(
    val billTitleTextFieldValue: String,
    val billValueTextFieldValue: String,
)

```